cost-analyzer/templates/cost-analyzer-network-costs-template.yaml

```diff
@@ -32,7 +39,7 @@ spec:
 {{- end }}
 {{- if .Values.networkCosts.resources }}
         resources:
-{{ toYaml .Values.networkCosts.resources | indent 10 }}
+          {{ toYaml .Values.networkCosts.resources | indent 10 }}
```

Comment:
Is this right? There's already an `indent` here.

Reply:
That's a good catch, going to blame auto format here, but my tests don't seem to generate something improperly formatted after running `helm template`. 

```yaml
      containers:
      - name: release-name-network-costs
        image: gcr.io/kubecost1/kubecost-network-costs:v15.0
        imagePullPolicy: Always
        resources:
          requests:
            cpu: 50m
            memory: 20Mi
          
        env:
        - name: NODE_NAME
```

I was pretty sure that this was going to be an issue, but both helm2 and helm3 generated the yaml with proper indention. Thoughts?

Reply:
Also worth noting that the `indent` command indents _all_ the output from the pipe source. In this case, all of the yaml gets indented by 4, but I would've still expected the first line to be indented further.

Reply:
Oh, huh. Well, all's well that ends well.